docs/self-managed/identity/user-guide/configuration/connect-to-an-existing-keycloak.md

```diff
@@ -17,32 +17,34 @@ In this guide, we'll demonstrate how to connect Identity to your existing Keyclo
 To connect Identity to an existing Keycloak instance, take the following steps:
 
 1. Log in to your Keycloak Admin Console.
-2. Select the realm you would like to connect Identity to. In our example, this is **Test Realm**.
+2. Select the realm you would like to connect Identity to. In our example, this is **camunda-platform**.
```

Comment:
FYI we are in the midst of renaming -- are we wanting to rename this to **camunda** @dlevy-camunda, or keep as-is?

Reply:
@christinaausley can you raise this in the [tmp-camunda-product-name](https://camunda.slack.com/archives/C05P016GR5M) channel? I'd like to see input from Mo and Max, not necessarily for this example but other programmatic examples. Basically how granular do we want to go with renaming?

Reply:
@christinaausley can you convert this to an issue and follow up? This shouldn't block this PR.